elixir/apps/web/lib/web/live/gateways_live/show.ex

```diff
@@ -136,7 +167,7 @@ defmodule Web.GatewaysLive.Show do
               OS version
             </th>
             <td class="px-6 py-4">
-              Linux 5.10.25-1-MANJARO x86_64
+              TODO: Linux 5.10.25-1-MANJARO x86_64
```

Comment:
I think we don't need to show this here if we have a properly-formatted User-Agent (which is required by our websocket protocol). Or if we want to keep it then we need to parse the user agent during rendering and present it here.

Reply:
Yeah, I wasn't sure about this one, so just left it in.  Happy to remove it if needed.

Reply:
This has been removed.